eclair-core/src/main/resources/reference.conf

```diff
@@ -547,6 +547,15 @@ eclair {
     enabled = true // enable automatic purges of expired invoices from the database
     interval = 24 hours // interval between expired invoice purges
   }
+
+  local-reputation {
```

Comment:
The name of this section is a bit confusing, this isn't for the reputation of our node, but rather for scoring our peers? I think it would make sense to:

- move this inside the `relay` section (it is only useful for payment relay)
- rename this section `peer-reputation`
- add a high-level comment to describe this section and link to the bLIP to let node operators find more resources

Reply:
Done. Except for the link to the blip, the blip says nothing about how to compute the reputation which is what this section is about.